### DISCUSSION TITLE
mp3tag not importing data from audible.com

### DISCUSSION THREAD

**Discussion by jimmy327** [seanap/Plex-Audiobook-Guide#26] [Q&A] [answered]
First, this is a great improvement over the way Plex handles audio books.  It's been needed for so long and I only wish I would have found this sooner.

I do have one issue though.  I've attempted to follow the guide for using mp3tag and integrating with Plex.  When I match to Audible.com it finds the book, and I can preview and it shows the correct data.  However, when it matches it is not importing any data.  I assume I am missing something, and probably not doing it correctly, however I've tried following the steps verbatim and it's not working. 

![prematch](https://user-images.githubusercontent.com/59419754/127958503-effcb475-5cae-4d6d-8b5a-7dd7abe2c8f6.JPG)

I can match the file successfully, but what I'm seeing afterward is 

![matched](https://user-images.githubusercontent.com/59419754/127958546-0e2afde6-46a7-4736-a4ad-d9c4f405c70d.JPG)

and the cover art and data is not updating.  I am using the most current version of mp3tag.  Could this be something with the way it is parsing data, or am I missing a step?

**Comment by seanap** [comment, marked as answer]
Hey, ~~you're doing everything right,~~ it does this sometimes when the webscraper errors. Is it for all books or just this one? Sometimes Audible does a site update, or weirdly codes certain book pages.

Edit: That book works on my end

**Reply by jimmy327** [reply]
Yes, it was doing it on almost every book that I pulled.  I imported 176 books.  The first, 10 or so maybe, worked, but then every single one failed after that, but I stopped doing the audible lookup after 15 or 20 and just used the metadata that was imported with them automatically.

I assumed I was doing something wrong at that point.  I'll give it another shot

**Reply by seanap** [reply]
Something is happening on your end, I am able to pull the book from your example
![image](https://user-images.githubusercontent.com/17012946/128789602-f28d5a0c-355b-4908-9f05-0fa645aef930.png)


Do your mp3's have the actual Track tags set?

Make sure the tracks are in the right order and select all the tracks, then do a Ctrl-K

**Reply by jimmy327** [reply]
Yeah, I have the track info.  Here's the process I'm going through. 

I have the book loaded, and select all, then press ctrl - K
![tag1](https://user-images.githubusercontent.com/59419754/128791326-e273b242-c045-49bc-b373-7fcf04e3200a.png)

this is the options I've been using
![tag2](https://user-images.githubusercontent.com/59419754/128791346-88f2dafb-82a8-42b6-975a-49b284456d85.png)

after it saves, I press ctrl-i to search
![tag3](https://user-images.githubusercontent.com/59419754/128791392-ea28a5c5-735a-4598-ba5f-cc61a89309b7.png)

and this is the result I get
![tag4](https://user-images.githubusercontent.com/59419754/128791402-33d85db5-77fc-4d95-8dd1-948e00253064.png)

What's interesting though, at the bottom there is the "This album on Audible.com"  When I click that, it loads the page for it.
![tag5](https://user-images.githubusercontent.com/59419754/128791442-127b011d-f034-4999-80f1-4ace7f14059e.png)

**Reply by jimmy327** [reply]
every book I've tried tonight is doing the same thing.  It's getting the right information, because I can click the link to go to the book's page on audible.  but it's not importing the data.   I'm starting to question if I didn't set something up right.

**Reply by jimmy327** [reply]
Nevermind.........

so i was on version 2.88, and it wasn't prompting me to update even though it's set to look for updates on startup.  Just updated to 3.08 and it loaded it up just fine.

so...it WAS on my end. 

sigh.....

**Reply by seanap** [reply]
I ran into the same thing myself, there was a small mention in the guide to "install/update to the latest version" but I should probably mention that it is incompatible with older versions.  Glad you got it!
